GentApp/Views/RegisterClientPage.xaml.cs

```diff
@@ -29,8 +15,8 @@ public RegisterClientPage()
             InitializeComponent();
 		}
 
-		void Register_To_LoginPage(object sender, RoutedEventArgs args) {
-			Frame.Navigate(typeof(LoginPage));
+		public void ToLoginPage(object sender, RoutedEventArgs args) {
+			var relayCommand = SimpleIoc.Default.GetInstance<UserViewModel>().ToLogin;
```

Comment:
SimpleIoc.Default.GetInstance<UserViewModel>().ToLogin;

Reply:
tmoet eigelijk SimpleIoc.Default.GetInstance<UserViewModel>().ToLogin.Execute(); zijn
want het is een command, das geen methode